src/app/utils/request.ts

```diff
@@ -1,5 +1,8 @@
 import { Request } from 'express'
 
 export const getRequestIp = (req: Request) => {
-  return req.get('cf-connecting-ip') || req.ip
+  if (req.get) {
```

Comment:
why do you need this? if it's only because functions might call it with anything other than a Request object, maybe add a TODO linking to #42 to remove it once TS migration is done?

Reply:
Sigh, it's based on controller tests that don't mock the request, meaning get is not a defined function :(. Will add TODO

Reply:
Fixed by mocking req.get in the failing tests instead